app/app.tsx

```diff
@@ -21,10 +21,9 @@ chrome.tabs.query(queryParams, tabs => {
   if (typeof id === "undefined" || typeof url === "undefined") return;
 
   MessageService.getEnabledStatus(id, (enabled: boolean) => {
```

Comment:
you are getting the status here. So use it and update the state. there is two ways to do this.
1. do a `store.dispatch()` action needed to update the state.
2. export the initial data and use `Object.assign` or `spread` operator to update the enabled state and pass it as initialState in `store()`.